src/workers/personalizedDigestEmail.ts

```diff
@@ -166,6 +192,55 @@ const worker: Worker = {
       return;
     }
 
+    const analyticsQueue = fastq.promise(
+      async (
+        data: {
+          user_id: string;
+          device_id: string;
+          experiment_id: string;
+          variation_id: number;
+        } & AnalyticsEvent,
+      ) => {
+        await sendAnalyticsEvent([data]);
+      },
+      1,
+    );
+
+    const growthbookClient = getUserGrowthBookInstace(user.id, {
+      enableDevMode: process.env.NODE_ENV !== 'production',
+      subscribeToChanges: false,
+      trackingCallback: async (experiment, result) => {
+        analyticsQueue.push({
+          event_name: 'allocation',
+          event_timestamp: new Date(),
+          user_id: user.id,
+          device_id: 'api',
+          experiment_id: experiment.key,
+          variation_id: result.variationId,
+        });
+      },
+    });
+
+    try {
+      await growthbookClient.loadFeatures();
+    } catch (error) {
+      logger.error(
+        { personalizedDigest, err: error },
+        'Failed to load features',
+      );
+    }
```

Comment:
Loading features manually, don't think there is a better option since each worker has a new context. Logging for now just so we can see what is happening in first few tries. 

Basically if this fails user would fallback to default value eg. control.

Reply:
We can load it in the entry point of the background worker. This way no one else will have to think about it

Reply:
Nice, like this? e6f1c814768021831e091610f69369bab2c61ac0